files/plir-openhim/default.json

```diff
@@ -3,6 +3,7 @@
   "minimumCoreVersion": "3.4.0",
   "protocol": "https",
```

Comment:
so **https://openhimcore.openmrs.org:443**  should redirect to   **http://namanga.openmrs.org:9000**

@cintiadr i hope this is a better alternative , instead of the container watching for certificates